Working prototype with validation code

seasidesparrow opened the pull request:
modified:   README.md
 	new file:   adsfulltextschema/Acknowledgements.json
 	new file:   adsfulltextschema/Affil.json
 	new file:   adsfulltextschema/AffilID.json
 	new file:   adsfulltextschema/BibRef.json
 	new file:   adsfulltextschema/Contrib.json
 	new file:   adsfulltextschema/Document.json
 	new file:   adsfulltextschema/Figure.json
 	new file:   adsfulltextschema/Footnote.json
 	new file:   adsfulltextschema/Funder.json
 	new file:   adsfulltextschema/IsbnType.json
 	new file:   adsfulltextschema/IssnType.json
 	new file:   adsfulltextschema/Keyword.json
 	new file:   adsfulltextschema/OpenAccess.json
 	new file:   adsfulltextschema/OtherContrib.json
 	new file:   adsfulltextschema/PersistentID.json
 	new file:   adsfulltextschema/PubDates.json
 	new file:   adsfulltextschema/Publication.json
 	new file:   adsfulltextschema/RecordData.json
 	new file:   adsfulltextschema/Reference.json
 	new file:   adsfulltextschema/Section.json
 	new file:   adsfulltextschema/Table.json
 	new file:   adsfulltextschema/Title.json
 	new file:   adsfulltextschema/__init__.py
 	new file:   dev/jats_parse.py
 	new file:   requirements.txt
 	new file:   setup.py
 	new file:   test_data/OA_jats_aa53501_24.json
 	new file:   test_data/OA_jats_apj_976_1_106.json
 	new file:   validation_example.py

seasidesparrow committed:
Working prototype with validation code
 	modified:   README.md
 	new file:   adsfulltextschema/Acknowledgements.json
 	new file:   adsfulltextschema/Affil.json
 	new file:   adsfulltextschema/AffilID.json
 	new file:   adsfulltextschema/BibRef.json
 	new file:   adsfulltextschema/Contrib.json
 	new file:   adsfulltextschema/Document.json
 	new file:   adsfulltextschema/Figure.json
 	new file:   adsfulltextschema/Footnote.json
 	new file:   adsfulltextschema/Funder.json
 	new file:   adsfulltextschema/IsbnType.json
 	new file:   adsfulltextschema/IssnType.json
 	new file:   adsfulltextschema/Keyword.json
 	new file:   adsfulltextschema/OpenAccess.json
 	new file:   adsfulltextschema/OtherContrib.json
 	new file:   adsfulltextschema/PersistentID.json
 	new file:   adsfulltextschema/PubDates.json
 	new file:   adsfulltextschema/Publication.json
 	new file:   adsfulltextschema/RecordData.json
 	new file:   adsfulltextschema/Reference.json
 	new file:   adsfulltextschema/Section.json
 	new file:   adsfulltextschema/Table.json
 	new file:   adsfulltextschema/Title.json
 	new file:   adsfulltextschema/__init__.py
 	new file:   dev/jats_parse.py
 	new file:   requirements.txt
 	new file:   setup.py
 	new file:   test_data/OA_jats_aa53501_24.json
 	new file:   test_data/OA_jats_apj_976_1_106.json
 	new file:   validation_example.py

seasidesparrow committed:
adds data-avail, supplementary info, multilingual
 	new file:   adsfulltextschema/DataAvailability.json
 	modified:   adsfulltextschema/Document.json
 	deleted:    test_data/OA_jats_aa53501_24.json
 	deleted:    test_data/OA_jats_apj_976_1_106.json

seasidesparrow committed:
Adding language field
 	modified:   adsfulltextschema/Section.json

kelockhart commented on adsfulltextschema/Acknowledgements.json:
```diff
@@ -0,0 +1,20 @@
+        "type": "string"
+      }
+    },
+    "title": {
```
What will the `title` field be used for?

kelockhart commented on adsfulltextschema/Acknowledgements.json:
```diff
@@ -0,0 +1,20 @@
+{
+  "title": "Acknowledgements",
```
Where will grant acknowledgements be held?

kelockhart commented on adsfulltextschema/Affil.json:
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "Affil",
```
Do we need to duplicate this from the main ingest_data_model?

kelockhart commented on adsfulltextschema/Contrib.json:
```diff
@@ -0,0 +1,66 @@
+{
+  "title": "Contrib",
```
Similar comment as for `affil` - I don't think we need to duplicate this from the ingest_data_model

kelockhart commented on adsfulltextschema/DataAvailability.json:
```diff
@@ -0,0 +1,17 @@
+{
+  "title": "DataAvailability",
```
We'll likely be mining this section for URLs and DOIs - should we add space to contain that info here?

kelockhart commented on adsfulltextschema/Footnote.json:
```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Footnote",
```
Similar to the data availability section - we'll likely be mining URLs from here, should we make space for those?

kelockhart commented on adsfulltextschema/Reference.json:
```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Reference",
```
Is this the output from the reference pipeline, or the input to it?

kelockhart commented on adsfulltextschema/Table.json:
```diff
@@ -0,0 +1,19 @@
+      "type": "string"
+    },
+    "data": {
+      "type": "string"
```
Do we want to hold parsed data in here? If so, this should probably be an object type instead of a string

kelockhart reviewed (commented):
Overall, looks pretty good, but I had a few comments/questions. One overarching comment, though, is that there's some duplication from the ingest_data_model. Unless there's a strong reason for the duplication, I'd lean towards removing duplicated sections and only include schema that's not represented elsewhere.

seasidesparrow committed:
commits before calling ingest as reference
 	modified:   adsfulltextschema/DataAvailability.json
 	modified:   adsfulltextschema/Document.json
 	modified:   adsfulltextschema/Section.json
 	new file:   test_data/aa53501-24.xml.json

seasidesparrow committed:
Imports ingest_data_model objects via id
 	deleted:    adsfulltextschema/Affil.json
 	deleted:    adsfulltextschema/AffilID.json
 	deleted:    adsfulltextschema/Contrib.json
 	modified:   adsfulltextschema/Document.json
 	deleted:    adsfulltextschema/Funder.json
 	deleted:    adsfulltextschema/IsbnType.json
 	deleted:    adsfulltextschema/IssnType.json
 	deleted:    adsfulltextschema/Keyword.json
 	deleted:    adsfulltextschema/OpenAccess.json
 	deleted:    adsfulltextschema/OtherContrib.json
 	deleted:    adsfulltextschema/PersistentID.json
 	deleted:    adsfulltextschema/PubDates.json
 	deleted:    adsfulltextschema/Publication.json
 	deleted:    adsfulltextschema/RecordData.json
 	modified:   adsfulltextschema/Section.json
 	deleted:    adsfulltextschema/Title.json

seasidesparrow committed:
updated to use ingest_data_model v1.0.13 (adsabs)
 	modified:   adsfulltextschema/Document.json
 	deleted:    dev/jats_parse.py
 	modified:   requirements.txt
 	modified:   test_data/aa53501-24.xml.json
 	modified:   validation_example.py

seasidesparrow committed:
updating to use ingest_data_model v1.0.13
 	modified:   adsfulltextschema/Document.json
 	modified:   adsfulltextschema/Section.json
 	new file:   dev/jats_parse.py

seasidesparrow committed:
fix url
 	modified:   adsfulltextschema/Document.json

seasidesparrow committed:
updates to ingest_data_model v1.1.0 (candidate)
 	modified:   adsfulltextschema/Document.json
 	deleted:    adsfulltextschema/Footnote.json
 	modified:   adsfulltextschema/Section.json
 	new file:   dev/12_kjkxxb-45-1-149.xml.json
 	modified:   dev/jats_parse.py
 	new file:   dev/requirements.txt
 	modified:   requirements.txt
 	new file:   test_data/11_kjkxxb-45-1-135.xml.json
 	modified:   test_data/aa53501-24.xml.json
